website/views/shared/footer.jade

```diff
@@ -63,8 +63,6 @@ footer.footer(ng-controller='FooterCtrl')
             a(target='_blank', href='http://devs.habitica.com')=env.t('devBlog') + ' - The Forge'
           li
             a(target='_blank', href='/apidoc')=env.t('APIv3')
-          li
-            a(target='_blank', href='/static/api-v2')=env.t('APIv2')
```

Comment:
This seems to be the only use of this localized string in the repo.  Can it be removed from settings.json?